package.json

```diff
@@ -32,6 +32,7 @@
     "@atlaskit/theme": "2.2.0",
     "@atlaskit/tooltip": "6.0.0",
     "autosize": "1.18.13",
+    "bowser": "1.9.1",
```

Comment:
Does package.lock need to be updated?

What's the plan going forward with browser detection in lib-jitsi-meet? It's not that long ago lib-jitsi-meet's browser detection was exposed publicly.

Reply:
For the package-lock:
For some reason for me the package-lock stayed the same. Which was very weird. I'll try to maybe remove it and do npm install to check if there will be any changes. I suppose some other dependency is using bowser maybe ...

About the plans for the browser detection in lib-jitsi-meet:
I don't have any news about it unfortunately. We still have to discuss creating a separate JS util repo and including RTCBrowserType there. Maybe using bowser within RTCBrowserType can be discussed as a part of it.

Reply:
I think it would be good to talk about what the plan is before going forward, maybe during a retro.